.github/workflows/_e2e.yaml

```diff
@@ -28,42 +28,22 @@ jobs:
       matrix:
         parallelRunnerId: ${{ fromJSON((fromJSON(inputs.matrix).parallelism == '4' && '[0, 1, 2, 3]') || '[0]') }}
     steps:
-      - name: "Print parameters"
-        id: eval-params
-        run: |
-          RUN_TYPE=""
-          RUNNER="${{ fromJSON(inputs.runnersByArch)[env.E2E_PARAM_ARCH] }}"
-          if [[ "$RUNNER" == "circleci" ]]; then
-            RUN_TYPE="circleci"
-          elif [[ "$RUNNER" != "" ]]; then
-            RUN_TYPE="github"
-          fi
-          echo "Running with: ${{ toJSON(inputs) }} ${{ toJSON(env) }}"
-          echo "run-type=$RUN_TYPE">> $GITHUB_OUTPUT
-      - name: "GitHub Actions: check out code"
-        if: steps.eval-params.outputs.run-type == 'github'
-        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+      - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
```

Comment:
We don't differentiate GitHub Actions and CircleCI any more, so don't need the "GitHub Actions: " prefix in the name.